algosdk/future/transaction.py

```diff
@@ -1158,6 +1305,7 @@ class ApplicationCallTxn(Transaction):
         accounts (list[string], optional): list of additional accounts involved in call
         foreign_apps (list[int], optional): list of other applications (identified by index) involved in call
         foreign_assets (list[int], optional): list of assets involved in call
+        extra_pages (int, optional): additional program space for supporting larger TEAL assembly program. A maximum of 3 extra pages is allowed. A page is 1024 bytes.
```

Comment:
I'd prefer we say "larger programs", and avoid "TEAL assembly".

Also, let's keep the limits out of this, so we don't have multiple places to update when they change. (For example, the foreign_apps and assets don't say what the limit is.)